blog/_posts/2018-09-16-$we-init-blog-post-2.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Our little infinity - RGSoC 2018!
+layout: post
+created_at: Sun Sept 16 2018
+permalink: blog/2018-08-03-$we-init-blog-post-2
```

Comment:
Hi @Rahmeen14 ! Thanks for the PR. Wonderful blogpost. I Loved it. Just a minor comment- permalink needs to be same as your filename [check the date]. So, please fix that.

On a side note, via supervisor calls I know that you have been also writing blog posts on your personal blogs. So, if you would want us to highlight those posts, we would be happy to do so as well. You can may be add links to those posts somewhere in this post, if you want to. :)

Reply:
Hi! I have updated the permalink, thank you for pointing that out!
About the blog posts, we're working on 2 detailed blogs for our bots and it'll be done by the mid of next week. We'd be more than happy to include those links here, if this can wait until next week?
Thanks! :D

Reply:
@Rahmeen14 That sounds good. We can publish other posts until then. Just comment here after you update it with links next week.

Reply:
@nerdyvaishali  I have added links to our blogs!

Reply:
Awesome @Rahmeen14 ! Thanks!